02_getting_started.asciidoc

```diff
@@ -36,33 +36,59 @@ If users decide to outsource some functionality to a third party they usually gi
+- Does this Lightning wallet store its own keys under user control (self-custody) or are the keys held be a third-party custodian?
+
+At the highest level of abstraction, question 1 and 3 are the most elementary ones. 
+From these two questions we can derive four possible categories. 
+We can place these four categories into a quadrant as seen in Table <<lnwallet-categories>>. 
+But remember that this is just one way of categorizing Lightning wallets.
 
 [[lnwallet-categories]]
-.Broad Categories of "Lightning Wallets"
+.Lightning Wallets Quadrant
 |===
-| Wallet Type          | LN Node        | Keystore/Custody | Technical Skill
-| Full Node & Wallet   | Full Node      | Non-Custodial    | High
-| Non-Custodial Wallet | 3rd-party node | Non-Custodial    | Medium
-| Custodial Wallet     | 3rd-party node | Custodial        | Low
+|                        | *Full LN Node*      | *3rd-party LN Node*    
+| *Self-Custody*         | Q1: High technical skill, least trust in 3rd parties, most permissionless | Q2: Below medium technical skills, below medium trust in 3rd parties, requires some permissions 
+| *Custodial*            | Q3: Above medium technical skills, above medium trust in 3rd parties, requires some permissions | Q4: Low technical skills, high trust in 3rd parties, least permissionless 
 |===
 
+Q3, quadrant 3, where a full LN node is used but the keys are held by a custodian is currently not common. 
+Future wallets from that quadrant would let a user worry about the operational aspects of their node, but then delegate the access to the keys to a third party which may use primarily cold storage.
+
 Lightning wallets can be installed on a variety of devices, including laptops, servers, and mobile devices. To run a full Lightning node (one that participates in "gossip" and creates its own map of the network for path finding and routing) you will need to use a server or desktop computer, as mobile devices and laptops are usually not powerful enough in terms of capacity, processing, battery life, and connectivity.
 
+The category "Third-party Ligntning Nodes" can again be subdivided into:
+
+- Lightweight: means that the Lightning Node is operated by a third party and that the wallet obtains the required information through an API that connects wallet and third-party LN node.
+- None: means that not only is the Lightning Node operated by a third party but most of the wallet is operated by a third party in the cloud such that the wallet does not even need to make calls on a Lightning node API.
+
+These subcategories are used in Table <<lnwallet-examples>>.
+
+Other terms that need explanation in Table <<lnwallet-examples>> in column "Bitcoin Node" are:
+
+- Neutrino: Neutrino is a protocol to get data from Bitcoin. The Bitcoin Node is run by a third-party and accessed via "neutrino".
```

Comment:
Can point to BIP 157+158. Neutrino is the name of a _specific_ implementation of the client-side/wallet portion of these BIPs. However overtime, people have come to colloquially refer an implementation of the BIPs as "neutrino".

Reply:
Good idea, that makes is better.

Done, see Commit # 3 in 6d3e4e37409994a2ebd2e1f35c53196a32d6ca4f